utils/vqm2blif/src/base/hash.cpp

```diff
@@ -106,10 +108,10 @@ insert_in_hash_table(struct s_hash **hash_table,
  * created.                                                                 */
 
     int i;
```

Comment:
Merge this. 
File an issue (or fix if you have time): change from this hand-written hash table to std::unordered_map